apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/Optional.kt

```diff
@@ -18,6 +19,14 @@ sealed class Optional<out V> {
   object Absent : Optional<Nothing>()
 
   companion object {
+    @JvmStatic
+    fun <T> absent(): Optional<T> = Absent
```

Comment:
The generated Java code references `Optional.absent()` and `Optional.fromNullable`. Were these methods removed deliberately? Can `Optional.Absent` be used from Java?

Reply:
> Were these methods removed deliberately?

`fromNullable` was renamed to `presentIfNotNull` to make it more explicit. 

> Can Optional.Absent be used from Java?

You can do 

```java
Optional optional = Optional.Absent.INSTANCE;
```

I can see how that's cumbersome though so `Optional.absent()` looks good to me 👍

Reply:
I'll get rid of `fromNullable` and expose `presentIfNotNull` as `@JvmStatic`.